### ISSUE TITLE
Possibly Wrong training set ? 

### ISSUE THREAD

**Issue by zafarmah92** [PhaelIshall/SemEval2020-Code#1]
Hi , 
I am using your code to reproduce the results for SemEval2020-Task4-Commonsense-Validation-and-Explanation. 

given the data from the correct [repository](https://github.com/wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation/tree/master/ALL%20data). 

Now when I run the given code it expects to have a **label** column in train.csv file, However, in the given train.csv file it don't exsits. 

would be great if you give an explanation

**Comment by PhaelIshall** [comment]
Hello. Try using the data from the [starting kit ](https://github.com/wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation/tree/master/starting_kit). I created a new file that contains the concatenated columns for [data (from data file)][label (from answers file)]

My inputs files follow the format: `sentence 1, sentence 2, label `

Unfortunately, I couldn't release the changes I made into a new file, but it's pretty straightforward, just put the data in the same file. Let me know if it still doesn't work!
